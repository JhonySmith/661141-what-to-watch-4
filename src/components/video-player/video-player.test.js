import React from "react";
import renderer from "react-test-renderer";

import VideoPlayer from "./video-player.jsx";

const movie = {
  previewImage: `https://upload.wikimedia.org/wikipedia/commons/f/f6/70mm_film_print_with_DTS_sound.jpg`,
  previewVideo: `https://download.blender.org/durian/trailer/sintel_trailer-480p.mp4`
};

describe(`Video player snap test`, () => {
  it(`Render video player`, () => {
    const tree = renderer
      .create(
          <VideoPlayer
            muted
            movie={movie}
            isPlaying={true}
          />, {
            createNodeMock: () => {
              return {};
            }
          }).toJSON();

    expect(tree).toMatchSnapshot();
  });
});

