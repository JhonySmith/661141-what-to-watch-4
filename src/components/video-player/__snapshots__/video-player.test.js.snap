// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Video player snap test Render video player 1`] = `
<video
  height="175"
  poster="https://upload.wikimedia.org/wikipedia/commons/f/f6/70mm_film_print_with_DTS_sound.jpg"
  type="video/mp4"
  width="280"
>
  Sorry, your browser doesnt support embedded videos.
</video>
`;
